apps/shinkai-desktop/src/pages/settings.tsx

```diff
@@ -548,6 +596,26 @@ const SettingsPage = () => {
                   <TextField field={field} label={t('ollama.version')} />
                 )}
               />
+              <FormField
```

Comment:
Would it make sense to follow the same save UX as in our settings? See shinkai identity for reference

Reply:
I thought about it, but a save button just to confirm a number feels unnecessary and an extra click.
Unlike the Shinkai identity feature, where a save button is useful for reviewing a text string before confirmation, simply setting a number doesn't require that extra step. The instant activation of experimental features via a toggle switch provides a good comparison.